pkgs/applications/editors/010-editor/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl, makeDesktopItem, autoPatchelfHook, copyDesktopItems, perlPackages, zlib, fontconfig, freetype, libxkbcommon, cups, ... }:
+
+let
+  extract = fetchFromGitHub {
```

Comment:
Maybe we want to move that into its own package.

Reply:
I don't think it has much value outside of this package.

[InstallJammer](https://en.wikipedia.org/wiki/InstallJammer) is deprecated since 2010 and I had never heard of it before. The unpacking script was in fact created to unpack another software product (NXP LPCXpresso SDK), but that was pretty much the single reference to InstallJammer unpacking on the web.

I'd say we split it off if someone has a decent use case for it (e.g. another package requiring it), other than that, I think it's a one-purpose script.